microya/src/androidTest/java/com/papershift/microya/supporting/UploadSuccessResponse.kt

```diff
@@ -0,0 +1,61 @@
+package com.papershift.microya.supporting
+
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class UploadSuccessResponse(
```

Comment:
I don't understand the naming here, isn't `ImgurResponse` already the success response? I'm confused by the naming ... are there 2 responses?